src/samples/java/com/box/sdk/samples/appusers/DocDetailsServlet.java

```diff
@@ -0,0 +1,58 @@
+package com.box.sdk.samples.appusers;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.box.sdk.BoxAPIConnection;
+import com.box.sdk.BoxFile;
+import com.box.sdk.samples.appusers.helpers.BoxHelper;
+
+
+public class DocDetailsServlet extends HttpServlet {
+    static final String[] EXTENSIONS = {"as", "as3", "asm", "bat", "c", "cc", "cmake", "cpp", "cs", "css", "csv", "cxx",
```

Comment:
done, although the url exceeds the 120 character limit set by check-style.   Not sure how to break a link up over multiple lines so I am leaving this check-style violation for the time being.